src/shapes/blender.cpp

```diff
@@ -0,0 +1,263 @@
+#include <mitsuba/render/mesh.h>
+#include <enoki/color.h>
+#include "blender_types.h"
```

Comment:
If this isn't used anywhere else, maybe could move the code of this header in this file?